javascript/javascript1/week2/student.js

```diff
@@ -0,0 +1,35 @@
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+  if (class07Students.length === 6 && studentName !== "Queen Margrethe II") {
```

Comment:
This line will give you trouble, try the following to see for yourself:
- add 6 studenst
- add the queen (now you have 7 studenst)
- add a 7th studen (that isn't the queen)

It is usually a good idea to use "greater than or equal to" because it is more restrictive than "exactly 6". It protects against situations exactly like this one.

Reply:
Done. Will keep in mind .